src/sentry/static/sentry/app/views/groupDetails/actions.jsx

```diff
@@ -285,37 +284,6 @@ const GroupDetailsActions = createReactClass({
             />
           </div>
         )}
-
-        {group.pluginActions.length > 1 && !orgFeatures.has('new-issue-ui') ? (
-          <div className="btn-group more">
-            <DropdownLink className="btn btn-default btn-sm" title={t('More')}>
-              {group.pluginActions.map((action, actionIdx) => {
-                return (
-                  <MenuItem key={actionIdx} href={action[1]}>
-                    {action[0]}
-                  </MenuItem>
-                );
-              })}
-            </DropdownLink>
-          </div>
-        ) : (
-          group.pluginActions.length !== 0 &&
-          !orgFeatures.has('new-issue-ui') &&
-          group.pluginActions.map((action, actionIdx) => {
-            return (
-              <div className="btn-group" key={actionIdx}>
-                <a className="btn btn-default btn-sm" href={action[1]}>
-                  {action[0]}
-                </a>
-              </div>
-            );
-          })
-        )}
-        {group.pluginIssues &&
-          !orgFeatures.has('new-issue-ui') &&
-          group.pluginIssues.map(plugin => {
-            return <IssuePluginActions key={plugin.slug} plugin={plugin} />;
-          })}
```

Comment:
Are we definitely able to completely remove these plugin buttons?

Reply:
i'm pretty sure we can, given `new-issue-ui` was i think on for everyone and these all have a `!orgFeatures.has('new-issue-ui')` check

but @MeredithAnya can you confirm?

Reply:
Ahh, you're right about the `orgFeatures.has('new-issue-ui')`. lgtm